compiler/rustc_session/src/options.rs

```diff
@@ -849,6 +880,10 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
         "exclude the pass number when dumping MIR (used in tests) (default: no)"),
     dump_mir_graphviz: bool = (false, parse_bool, [UNTRACKED],
         "in addition to `.mir` files, create graphviz `.dot` files (default: no)"),
+    dump_mir_spanview: Option<MirSpanview> = (None, parse_mir_spanview, [UNTRACKED],
+        "in addition to `.mir` files, create `.html` files to view spans for \
+        all `statement`s (including terminators), only `terminator` spans, or \
+        spans that encompass all spans, for each basic `block`."),
```

Comment:
maybe a clearer way to say this is "or complete coverage (`block`)"?

Reply:
That sounds misleading because `statement` is "complete coverage". The `block` span is a single span per block, starting at the lowest lo() and ending at the highest hi() of all of that block's statements and its terminator. So it loses a lot of fidelity.

How about: 

or \
        computed `block` spans (one span encompassing a block's terminator and \
        all statements).

Reply:
Oh, I misunderstood then. "or spans that encompass all spans" is confusing to me. What you put in parentheses seems perfect.

Reply:
I changed it to:

```rust
    dump_mir_spanview: Option<MirSpanview> = (None, parse_mir_spanview, [UNTRACKED],
        "in addition to `.mir` files, create `.html` files to view spans for \
        all `statement`s (including terminators), only `terminator` spans, or \
        computed `block` spans (one span encompassing a block's terminator and \
        all statements)."),
```